Opcodes/CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ find_library(WIIUSE_LIB wiiuse)
 check_include_file(p5glove.h P5GLOVE_H)
 find_library(P5GLOVE_LIB p5glove)
 
-if(BUILD_FAUST_OPCODES)
+if(BUILD_FAUST_OPCODES AND FAUST_FOUND)
```

Comment:
Use `check_deps(BUILD_FAUST_OPCODES FAUST_FOUND)` please. This gives a message to users when it is disabled, and can be made to fail with `-DFAIL_MISSING=ON` for distribution builds.

Reply:
Will add!